openblas/PKGBUILD

```diff
@@ -0,0 +1,105 @@
+# Maintainer: Felix Yan <felixonmars@archlinux.org>
+# Contributor: Giuseppe Borzi <gborzi _AT_ ieee _DOT_ org>
+
+pkgbase=openblas
+pkgname=(openblas openblas64 blas-openblas blas64-openblas)
+_pkgname=OpenBLAS
+pkgver=0.3.26
+pkgrel=3
+_blasver=3.12.0
+pkgdesc="An optimized BLAS library based on GotoBLAS2 1.13 BSD"
+arch=('x86_64')
+url="https://www.openblas.net/"
+license=('BSD')
+depends=('gcc-libs')
+makedepends=('cmake' 'perl' 'gcc-fortran')
+source=(${_pkgname}-v${pkgver}.tar.gz::https://github.com/xianyi/OpenBLAS/archive/v${pkgver}.tar.gz)
+sha512sums=('01d3a536fbfa62f276fd6b1ad0e218fb3d91f41545fc83ddc74979fa26372d8389f0baa20334badfe0adacd77bd944c50a47ac920577373fcc1d495553084373')
+
+build() {
+  # Setting FC manually to avoid picking up f95 and breaking the cmake build
+  # https://github.com/xianyi/OpenBLAS/issues/4072#issuecomment-1576388332
+
+  # Setting ASM flags for CET support.  Setting FFLAGS for CET support.
+  # Remove ` -Wformat -Werror=format-security` not supported by gcc-fortran.
+
+  ASMFLAGS=$CFLAGS FFLAGS=${CFLAGS/ -Wformat -Werror=format-security/} FC=gfortran cmake -B build -S $_pkgname-$pkgver \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DBUILD_SHARED_LIBS=ON \
+    -DBUILD_TESTING=OFF \
+    -DNO_AFFINITY=ON \
+    -DUSE_OPENMP=1 \
+    -DNO_WARMUP=1 \
+    -DTARGET=HASWELL \
```

Comment:
Consider to put another PKGBUILD for the v4 repos, which enables SKYLAKE-X.
Something like this as directory structure (pkgname stays same):

openblas/openblas-v3
openblas/openblas-v4